src/wdio/services/AxeService.js

```diff
@@ -11,7 +11,6 @@ export default class AxeService {
   // eslint-disable-next-line class-methods-use-this
   before(config) {
```

Comment:
I think i see why you are putting config in capabilities... this should probably be  `onPrepare`. You can then use the config param to pull the axe config so you don't have to put it in capabilities.

```js
    /**
     * Gets executed once before all workers get launched.
     * @param {Object} config wdio configuration object
     * @param {Array.<Object>} capabilities list of capabilities details
     */
    onPrepare: function (config, capabilities) {
    },
```

Reply:
Moving to using onPrepare does not allow us to have access to the global browser variable.  
Using before is the earliest we can have access to the browser variable, which before only has the two params of the capabilities and the specs.